Chinese/mwe-chi.tsv

```diff
@@ -99,7 +99,6 @@
 一_num 次_msr 酿造_verb 的_part 量_noun	N5
 一_num 步_msr 一_num 步_msr 地_noun	A5.1+ A12-
 一_num 水盆_noun	O2/O1.2
-一_num 法郎_msr N3.1/I1
 一_num 法郎_msr 货币_noun	I1	N3.1/I1

```

Comment:
I think the `test_collection.py` script needs to be stricter and only allow fields/columns with header names, and if the field contains no header name then it should raise a `ValueError`. This is needed as we can see in this highlighted line that some of the semantic tags have been separated through the use of a tab instead of white space, which needs correcting.

Reply:
Sorry I was wrong it should be line 103, `一_num 法郎_msr 货币_noun	I1	N3.1/I1`, not the highlighted line.

Reply:
I think both lines 102 and 103 should be kept ... what's the reason to delete them?

Reply:
I agree lines 102 and 103 should be kept, however the formatting on line 103 should be changed as currently it is `一_num 法郎_msr 货币_noun\tI1\tN3.1/I1`, whereby `\t` represents a tab, when it should be `一_num 法郎_msr 货币_noun\tI1\sN3.1/I1`, whereby `\s` represents a single white space.

Reply:
This should now be resolved since commit `6133b74`